store/index.js

```diff
@@ -936,5 +941,14 @@ export const mutations = {
 
         // 2. Save the updated annotation in the store
         p_state.annotationCount = count;
-    }
+    },
+
+    resetState(p_state) {
+        Object.keys(p_state).forEach((key) => {
+          const initialState = getDefaultState();
+          if (Object.prototype.hasOwnProperty.call(initialState, key)) {
```

Comment:
```suggestion
          if (Object.hasOwn(initialState, key)) {
```

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn and we don't plan on supporting older browsers

We could most likely also just go with `initialState.hasOwnProperty(key)` here because I think your version mainly guards against the possibility of `"hasOwnProperty"` being an actual key in the store - which we don't have.

Reply:
Tried it, didn't work.